### ISSUE TITLE
Debug: scrollbar arrow can't be clicked on in GDB terminal

### ISSUE THREAD

**Issue by ma-michelle** [cee-studio/IDE#12]
1. demo: https://cee.studio/?bucket=200730-uvW&name=priority-queue.LgVM
2. breakpoint should be set on line 61
3. debug mode > GDB console

If you try pressing on the 'up' button on the scrollbar it doesn't work, however scrolling with the mouse does.  

![image](https://user-images.githubusercontent.com/56993432/88880542-dcc58380-d1e1-11ea-8f94-62d1151d5108.png)

The 'up' button does work on the 'build' tab, however.

![image](https://user-images.githubusercontent.com/56993432/88880511-c4556900-d1e1-11ea-8a1d-799635deae60.png)

**Comment by mlite** [comment]
can you please verify this bug?

**Comment by bchiu3** [comment]
Yes, this is actually a bug. Interestingly though, once you resize the window of the IDE, the bug disappears. The bug also disappears if you go to any other tab, click it, and go back to GDB console.
